functions/src/types.ts

```diff
@@ -36,3 +37,37 @@ export type Message = {
   timestamp: number
   image?: { caption: string; id: string; mime_type: string }
 }
+
+export type Checker = {
+  name: string
+  type: "human" | "ai"
+  isActive: boolean | null
+  isOnboardingComplete: boolean | null
+  singpassOpenId: string | null
+  telegramId: string | null
+  whatsappId?: string | null
+  level: number
+  experience: number
+  numVoted: number
+  voteWeight: number
+  numCorrectVotes: number
+  numVerifiedLinks: number
+  preferredPlatform: string | null
+  lastVotedTimestamp: Timestamp | null
+  getNameMessageId: string | null
+}
+
+export type VoteRequest = {
+  factCheckerDocRef: DocumentReference
+  platformId: string | null
+  platform: "whatsapp" | "telegram" | "agent"
+  hasAgreed: boolean | null
+  triggerL2Vote: boolean | null
+  triggerL2Others: boolean | null
+  sentMessageId: string | null
+  vote: 0 | 1 | 2 | 3 | 4 | 5 | null
+  category: string | null
+  createdTimestamp: Timestamp | null
+  acceptedTimestamp: Timestamp | null
+  votedTimestamp: Timestamp | null
+}
```

Comment:
Can I confirm my understanding that these fields are what we expect the new database schema to be conforming to?